internal/server/email/smtp.go

```diff
@@ -106,6 +106,9 @@ func SendSMTP(msg Message) error {
 		return err
 	}
 
+	if err := writeln(w, `X-SMTPAPI: {"filters":{"bypass_list_management":{"settings":{"enable":1}}}}`); err != nil {
```

Comment:
This is currently in a function called `SendSMTP`, but I think this setting is only appropriate for transactional emails. I think it would be easy for us to forget about this and accidentally use this function for emails that should not bypass these lists.

Could we have the caller enabled this with a setting on `Message`, so that the default is to use all the lists? Could be something like `Transactional bool`, or more specific like `ByPassListManagement bool`.

It'd also be great to link to the docs for this setting. If you add a field to `Message` to enable this, that'd be a great place for the comment with the link.